### ISSUE TITLE
Refresh page, slider show the last item at the first in mode: 'horizontal'

### ISSUE THREAD

**Issue by yujiangshui** [stevenwanderski/bxslider-4#694]
The first time open the page, it will be fine. But then refresh the page, slider will jump from the really first item to the last item clone DOM.

If you have the `auto: true` option, next change will go to the second item acrossing the first item from the last one, then everything will be fine.

The [official example](http://bxslider.com/examples/auto-show-start-stop-controls) also has the problem mentioned above, does anyone have any solution

**Comment by kellykarnetsky** [comment]
I noticed this issue as well and it seems to be related to Chrome only. It looks like it's random, maybe a race condition in Chrome. In Firefox and IE, I'm not having the same issue. 

Resizing the window seems to fix it so I took a look at the `resizeWindow`. There is a conditional that only does the resize if the window size is different from the initial size so what I did was change ~110 where `windowWidth` and `windowHeight` were set as the width and height and just defined them as empty so it looks like `var windowWidth, windowHeight;`. 

This allows the resize to fire once fixing the Chrome and Firefox bug.

I don't think this is the best solution but I wanted to give a little more info to help resolve this.
